src/includes/getting-started-install/flutter.mdx

```diff
@@ -1,3 +1,5 @@
+Get the SDK from [pub.dev](https://pub.dev/packages/sentry_flutter) by adding the following to your `pubspec.yaml`:
+
 ```yml {filename:pubspec.yaml}
 dependencies:
   sentry_flutter: ^4.0.4
```

Comment:
not only here, but also the Dart SDK, now we support the release-registry, so the version can be picked up automatically.
See https://github.com/getsentry/sentry-release-registry/pull/46

Reply:
@bruno-garcia fixed it here btw #3193

Reply:
I will delete this .